build.rs

```diff
@@ -2,22 +2,35 @@ use std::process::Command;
 
 // Example custom build script.
 fn main() {
+    // TODO: use handlebars to automatically generate filters, that way you abstract away the code that is used
+    //       for all filters
+
     // Note: needs to be run with cargo -vv to generate output from these commands.
     eprintln!("{:?}",
-              Command::new("rustc").args(&["plugins/sample.rs", "--crate-type=dylib"]).output().unwrap());
+              Command::new("rustc").args(&["plugins/sample_filter.rs", "--crate-type=dylib"]).output().unwrap());
+    /*
```

Comment:
Why are there commented lines of code here? Let's remove them if they are no longer needed.

Reply:
Good call.  This is fixed now.